Source/WebCore/Modules/webauthn/fido/FidoConstants.h

```diff
@@ -277,6 +277,29 @@ const int64_t kCtapGetAssertionRequestOptionsKey = 5;
 const int64_t kCtapGetAssertionPinUvAuthParamKey = 6;
 const int64_t kCtapGetAssertionPinUvAuthProtocolKey = 7;
 
+const int64_t kCtapAuthenticatorGetInfoVersionsKey = 0x01;
+const int64_t kCtapAuthenticatorGetInfoExtensionsKey = 0x02;
+const int64_t kCtapAuthenticatorGetInfoAAGUIDKey = 0x03;
+const int64_t kCtapAuthenticatorGetInfoOptionsKey = 0x04;
+const int64_t kCtapAuthenticatorGetInfoMaxMsgSizeKey = 0x05;
+const int64_t kCtapAuthenticatorGetInfoPinUVAuthProtocolsKey = 0x06;
+const int64_t kCtapAuthenticatorGetInfoMaxCredentialCountInListKey = 0x07;
+const int64_t kCtapAuthenticatorGetInfoMaxCredentialIdLengthKey = 0x08;
+const int64_t kCtapAuthenticatorGetInfoTransportsKey = 0x09;
+const int64_t kCtapAuthenticatorGetInfoAlgorithmsKey = 0x0a;
+const int64_t kCtapAuthenticatorGetInfoMaxSerializedLargeBlobArrayKey = 0x0b;
+const int64_t kCtapAuthenticatorGetInfoForcePINChangeKey = 0x0c;
+const int64_t kCtapAuthenticatorGetInfoMinPINLengthKey = 0x0d;
+const int64_t kCtapAuthenticatorGetInfoFirmwareVersionKey = 0x0e;
+const int64_t kCtapAuthenticatorGetInfoMaxCredBlobLengthKey = 0x0f;
+const int64_t kCtapAuthenticatorGetInfoMaxRPIDsForSetMinPINLengthKey = 0x10;
+const int64_t kCtapAuthenticatorGetInfoPreferredPlatformUvAttemptsKey = 0x11;
+const int64_t kCtapAuthenticatorGetInfoUVModalitysKey = 0x12;
+const int64_t kCtapAuthenticatorGetInfoCertificationsKey = 0x13;
+const int64_t kCtapAuthenticatorGetInfoRemainingDiscoverableCredentialsKey = 0x14;
+const int64_t kCtapAuthenticatorGetInfoVendorPrototypeConfigCommandsKey = 0x15;
```

Comment:
nit - these could all use `constexpr`, but I see these just match the existing ones.